packages/ember-glimmer/tests/integration/components/curly-components-test.js

```diff
@@ -2782,6 +2782,54 @@ moduleFor('Components test: curly components', class extends RenderingTest {
     }, /didInitAttrs called/);
   }
 
+  ['@test did{Init,Receive}Attrs fires after .init() but before finishChains'](assert) {
```

Comment:
Can you tweak this to avoid the "inside baseball" of `finishChains`?  Would you be ok with something like:

`@test did{Init,Receive}Attrs fires after .init() but before events are triggered and observers are setup`

I fear using `finishChains` will lead to confusion about what this test is actually intended to test over time.